### ISSUE TITLE
Job namespace selector rendered too small

### ISSUE THREAD

**Issue by optiz0r** [hashicorp/nomad#10769]
### Nomad version

1.1.1+ent

### Issue

The job namespace selector dropdown sometimes renders too small. So small, that you can't see the "Namespace" label, or any of the values within. Once selecting a namespace, on page refresh it's rendered wide enough to be usable. This might be related to an ACL token not having any permissions within the default namespace.

Examples:

collapsed:
![image](https://user-images.githubusercontent.com/1030831/122236925-d3efa600-ceb6-11eb-9235-fdbdcf1ae5d4.png)

expanded:
![image](https://user-images.githubusercontent.com/1030831/122236559-883cfc80-ceb6-11eb-8590-8b07906ee944.png)

### Reproduction steps

* Create an ACL token which has no access to default namespace, but has access to at least one other namespace
* Navigate to the jobs ui view
* Observes the namespace dropdown is rendered poorly

#### Expected Result

Namespace dropdown to always be rendered at least wide enough to see the Namespace label, so you can see what the field is for.

#### Actual Result

Rendered so narrow, that it becomes hard to tell it's actually somthing you can interact with.

**Comment by tgross** [comment]
Thanks @optiz0r. I think this is a dupe of https://github.com/hashicorp/nomad/issues/10566, but because #10566 covers a few other related papercut behaviors we'll keep this open to make it easier for other folks reporting issues to search it.

**Comment by optiz0r** [comment]
You're right, I did a quick search before opening the issue, but didn't spot that one. I agree they're very similar. Don't mind if you want to keep this open, or close and just note the reproduction steps on #10566.
